.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -4,36 +4,17 @@
 name: Test Runner Nightly Integration Tests
 
 on:
-  push:
-    branches: [ dev ]
-  pull_request:
-      branches: [ '**' ]
+  workflow_dispatch: {}
+  schedule:
+  - cron: '0 7 * * *' # run at 7 AM UTC, 2 AM ET.
```

Comment:
Right now, results show up in the repo's Actions page. I think we want to surface nightly failures somehow, but I'm not sure what we currently do beyond uploading results - @ungwudik or @ansingh7115, do you have any more context on surfacing nightly test failures?

Reply:
Ultimately, we want to create a central dashboard for all services. This involves rethinking the different dashboards we currently have and possibly merging them. This will be a big effort and QA hasn't worked through all of it yet. In the meantime, we should post WSM nightly test results to #dsde-qa Slack channel.